cmd/gke-certificates-controller/app/gke_approver.go

```diff
@@ -66,15 +69,18 @@ func (a *gkeApprover) handle(csr *capi.CertificateSigningRequest) error {
 		if !r.recognize(a.opts, csr, x509cr) {
 			continue
 		}
-		tried = append(tried, r.permission.Subresource)
+		glog.Infof("validator %q matched CSR %q", r.name, csr.Name)
+		tried = append(tried, r.name)
 		if r.validate != nil && !r.validate(a.opts, csr, x509cr) {
+			glog.Infof("validator %q denied CSR %q", r.name, csr.Name)
 			return a.updateCSR(csr, false, r.denyMsg)
 		}
 
 		approved, err := a.authorizeSAR(csr, r.permission)
 		if err != nil {
 			return err
 		}
+		glog.Infof("validator %q SAR approval CSR %q: %v", r.name, csr.Name, approved)
```

Comment:
I'm not sure what this comment means.

Reply:
Split into two comments, hopefully it's a bit clearer